release/artifacts.yaml

```diff
@@ -1,230 +1,196 @@
+  # This is a tarball of the fat jar.
+  mavenUpload: false
 - target: //extractor:extractor-binary
   type: jar-deploy
+  # This is a fat jar.
+  mavenUpload: false
 - target: //ledger-api/grpc-definitions:ledger-api-scalapb
   type: jar-scala
-  mavenUpload: true
 - target: //ledger-api/testing-utils:testing-utils
   type: jar-scala
-  mavenUpload: true
 - target: //language-support/scala/bindings:bindings
   type: jar-scala
-  mavenUpload: true
 - target: //ledger-api/rs-grpc-akka:rs-grpc-akka
   type: jar-scala
-  mavenUpload: true
 - target: //ledger-api/rs-grpc-testing-utils:rs-grpc-testing-utils
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-akka:ledger-api-akka
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-client:ledger-api-client
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-domain:ledger-api-domain
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-common:ledger-api-common
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-auth:ledger-api-auth
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-auth-client:ledger-api-auth-client
   type: jar-lib
-  mavenUpload: true
 - target: //ledger/ledger-api-health:ledger-api-health
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/sandbox:sandbox
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/ledger-api-test-tool:ledger-api-test-tool
   type: jar-deploy
-  mavenUpload: true
 - target: //language-support/scala/codegen:codegen
   type: jar-scala
-  mavenUpload: true
 - target: //language-support/scala/codegen:codegen-main
   type: jar
+  # This is a fat jar.
+  mavenUpload: false
 - target: //language-support/scala/bindings-akka:bindings-akka
   type: jar-scala
-  mavenUpload: true
 - target: //language-support/java/codegen:shaded_binary
   type: jar-scala
-  mavenUpload: true
 - target: //language-support/java/codegen:lib
   type: jar-scala
-  mavenUpload: true
 - target: //navigator/backend:navigator-binary
   type: jar-deploy
+  # This is a fat jar.
+  mavenUpload: false
 - target: //language-support/codegen-common:codegen-common
   type: jar-scala
-  mavenUpload: true
 - target: //language-support/codegen-main:shaded_binary
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/participant-state:participant-state
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/participant-state-index:participant-state-index
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/sandbox:ledger-api-server
   type: jar-scala
-  mavenUpload: true
 - target: //ledger/participant-state/protobuf:ledger_configuration_java_proto
   type: jar-proto
-  mavenUpload: true
 - target: //ledger/participant-state/kvutils:daml_kvutils_java_proto
   type: jar-proto
-  mavenUpload: true
 - target: //ledger/participant-state/kvutils:kvutils
   type: jar-scala
-  mavenUpload: true
 - target: //ledger-service/lf-value-json:lf-value-json
```

Comment:
This means`lf-value-json`, `utils`, `db-backend` and `http-json` are now uploaded. Happy to change any of that. Afaict this is the only change.